test/config-specs.js

```diff
@@ -64,17 +64,21 @@ describe('Config', () => {
       process = _process;
     });
     describe('checkNodeOk', () => {
-      it('should fail if node is below 0.10', () => {
+      it('should fail if node is below 0.12', () => {
         process.version = 'v0.9.12';
-        checkNodeOk.should.throw;
+        checkNodeOk.should.throw();
+        process.version = 'v0.1';
```

Comment:
Please let me know if you'd prefer to separate these new assertions into separate test cases.